code/datums/uplink_items/uplink_general.dm

```diff
@@ -274,12 +274,9 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	name = "Fiber Wire Garrote"
 	desc = "A length of fiber wire between two wooden handles, perfect for the discrete assassin. This weapon, when used on a target from behind \
 			will instantly put them in your grasp and silence them, as well as causing rapid suffocation. Does not work on those who do not need to breathe."
-	reference = "GAR"
-	item = /obj/item/twohanded/garrote
+	item = /obj/item/garrote
```

Comment:
You removed the reference here on accident I think

Reply:
oopsie daisy